public/app/features/plugins/admin/components/PluginListCard.tsx

```diff
@@ -26,10 +26,10 @@ export function PluginListCard({ plugin, pathName }: PluginListCardProps) {
             className={styles.image}
             height={LOGO_SIZE}
           />
-          <h3 className={styles.name}>{plugin.name}</h3>
+          <h2 className={styles.name}>{plugin.name}</h2>
           {plugin.type && (
-            <div className={styles.icon}>
-              <Icon name={IconName[plugin.type]} aria-label={`${plugin.type} plugin icon`} />
+            <div className={styles.icon} data-testid={`${plugin.type} plugin icon`}>
+              <Icon name={IconName[plugin.type]} />
```

Comment:
**Question:** why did it not like the `area-label` on this one?

Reply:
The `<Icon />` adds the `aria-label` to a `<div />` which isn't a valid element for the attribute. https://www.tpgi.com/short-note-on-aria-label-aria-labelledby-and-aria-describedby/

Reply:
Thanks mate 👍